sonar-batch/src/main/java/org/sonar/batch/bootstrap/BatchWsClientProvider.java

```diff
@@ -48,15 +47,15 @@ public synchronized WsClient provide(final GlobalProperties settings, final Envi
       // TODO proxy
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)